mono/mini/debugger-engine.h

```diff
@@ -189,6 +189,34 @@ typedef struct {
 	int nframes;
 } SingleStepArgs;
 
+/*
+ * OBJECT IDS
+ */
+
+/*
+ * Represents an object accessible by the debugger client.
+ */
+typedef struct {
+	/* Unique id used in the wire protocol to refer to objects */
+	int id;
+	/*
+	 * A weakref gc handle pointing to the object. The gc handle is used to 
+	 * detect if the object was garbage collected.
+	 */
+	guint32 handle;
+} ObjRef;
+
+
+static void
+free_objref (gpointer value)
```

Comment:
This function shouldn't be a header file, please move it back to the `.c` file. Otherwise this function is generated in each module that includes this header.

Reply:
Thanks @lewurm , it is done here:
https://github.com/mono/mono/pull/12733